packages/react/src/Image/Image.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import clsx from 'clsx'
+import styles from './Image.module.css'
+
+type AspectRatio = '1:1' | '16:9' | '16:10' | '4:3' | 'custom'
+
+export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  ref?: React.Ref<HTMLImageElement>
+  alt: string
+  as?: 'picture' | 'img'
+  aspectRatio?: AspectRatio
+}
+
+const aspectRatioResolver = (ratio?: AspectRatio) => {
+  if (ratio === undefined) return ''
+  if (typeof ratio === 'string') {
+    if (ratio === 'custom') return 'custom'
+    const [width, height] = ratio.split(':').map(Number)
+    return `${width}-${height}`
+  }
+}
+
+export const Image = ({className, aspectRatio, as = 'img', ref, alt, width, height, ...rest}: ImageProps) => {
+  if (as === 'picture') {
+    return (
+      <picture
+        className={clsx(styles['Image-container'], aspectRatio && styles[`Image--${aspectRatioResolver(aspectRatio)}`])}
+      >
+        <img
+          ref={ref}
+          alt={alt}
+          width={width ? width : '100%'}
+          height={width ? width : '100%'}
+          className={clsx(styles.Image, className)}
+          {...rest}
+        />
+      </picture>
+    )
+  }
+  if (aspectRatio) {
+    return (
+      <span className={clsx(styles['Image-container'], styles[`Image--${aspectRatioResolver(aspectRatio)}`])}>
+        <img
+          ref={ref}
+          alt={alt}
+          width={width ? width : '100%'}
+          height={width ? width : '100%'}
+          className={clsx(styles.Image, className)}
+          {...rest}
+        />
+      </span>
+    )
+  }
+  return (
+    <img
+      ref={ref}
+      alt={alt}
+      className={clsx(styles.Image, className)}
+      width={width ? width : 'initial'}
```

Comment:
Out of curiousity why do we need to set `initial` in normal usage of this component? If it were to behave like a normal `img` out-of-box, could we make this undefined instead?

![Screenshot 2023-04-14 at 12 43 58](https://user-images.githubusercontent.com/13340707/232035267-65064e91-e2c5-4b3c-b9d7-03836bd08953.png)

Reply:
Reopening this as it still seems to be happening.. can those be removed @JoshBowdenConcepts?
![Screenshot 2023-04-17 at 16 51 33](https://user-images.githubusercontent.com/13340707/232541902-9d415234-8c01-4f78-9119-4f89db4594dc.png)

Reply:
Whoops! I didn't think I closed it but it should be updated now!